src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/text/TextEditingTarget.java

```diff
@@ -3533,14 +3526,11 @@ public void execute()
                                            range.getStart().getColumn()),
                      true);
             }
-            docDisplay_.setSelection(
```

Comment:
I think that executeNextChunk might still want to be setting the selection to the next chunk (but perhaps that can be done up in onExecuteNextChunk rather than in here).